### PR TITLE
Add RV64 + Clean-up crt0 generation

### PR DESCRIPTION
**Pull request by gmbr3**

### CODE REVIEW

File: efi/meson.build [hunk 1]
```diff
@@ -222,26 +222,11 @@ endif
 if objcopy_manualsymbols
   # older objcopy for Aarch64 and ARM32 are not EFI capable.
```
Comment (gmbr3) [inline efi/meson.build:223]: Needs updating

Comment (gmbr3) [reply efi/meson.build]: done

**Review by superm1 — APPROVED** [review]

**Comment by superm1** [comment]
Is this conceptual or actually validated working?

If it's validated working in think we should do some checks on other architecture as well and tag a new release.

**Comment by gmbr3** [comment]
It at least loads in virtual RISC-V UEFI
[qemu-ovmf-fwupd.txt](https://github.com/fwupd/fwupd-efi/files/14323506/qemu-ovmf-fwupd.txt)
[rv64.txt](https://github.com/fwupd/fwupd-efi/files/14323507/rv64.txt)

**Comment by superm1** [comment]
> It at least loads in virtual RISC-V UEFI
> [qemu-ovmf-fwupd.txt](https://github.com/fwupd/fwupd-efi/files/14323506/qemu-ovmf-fwupd.txt)
> [rv64.txt](https://github.com/fwupd/fwupd-efi/files/14323507/rv64.txt)
> 

Cool thanks. Can you see if anything is needed on the fwupd side as well? There should be some things.

**Comment by superm1** [comment]
I opened up https://github.com/fwupd/fwupd/pull/6814 as I know it's a baseline requirement, please open anything else you find though in case there is more.

**Pull request merged**